stardis/tests/stardis_test_config.yml

```diff
@@ -5,16 +5,14 @@ model:
     fname: docs/quickstart/sun.mod
     final_atomic_number: 5
 opacity:
-    file:
-        Hminus_bf: docs/quickstart/h_minus_cross_section_wbr.dat
     bf:
         H_I: {}
     ff:
         H_I: {}
-    disable_electron_scattering: False
+    disable_electron_scattering: True
```

Comment:
Why is e scattering being disabled?

Reply:
Because it's covered in another test config. This change was just to broaden test coverage and test when it's also off.